Stratis.Bitcoin.Features.Wallet.Tests/WalletManagerTest.cs

```diff
@@ -1242,7 +1242,7 @@ public void GetSpendableTransactionsWithChainOfHeightZeroReturnsNoTransactions()
 
             walletManager.Wallets.Add(wallet);
 
-            var result = walletManager.GetSpendableTransactionsInWallet("myWallet", confirmations: 0);
+            var result = walletManager.GetSpendableTransactionsInWallet("myWallet", confirmations: 1);
```

Comment:
no appreciation of all these vars everywhere ... but i've understood that code and comment policy is not for tests (why oh why), so whatever :)